docs/en-US/component/time-picker.md

```diff
@@ -49,24 +49,24 @@ time-picker/range
 | readonly              | whether TimePicker is read only                          | ^[boolean]                                                                                      | false       |
 | disabled              | whether TimePicker is disabled                           | ^[boolean]                                                                                      | false       |
 | editable              | whether the input is editable                            | ^[boolean]                                                                                      | true        |
 | clearable             | whether to show clear button                             | ^[boolean]                                                                                      | true        |
 | size                  | size of Input                                            | ^[enum]`'large' \| 'default' \| 'small'`                                                        | —           |
-| placeholder           | placeholder in non-range mode                            | ^[string]                                                                                       | —           |
+| placeholder           | placeholder in non-range mode                            | ^[string]                                                                                       | ''          |
 | start-placeholder     | placeholder for the start time in range mode             | ^[string]                                                                                       | —           |
 | end-placeholder       | placeholder for the end time in range mode               | ^[string]                                                                                       | —           |
 | is-range              | whether to pick a time range                             | ^[boolean]                                                                                      | false       |
 | arrow-control         | whether to pick time using arrow buttons                 | ^[boolean]                                                                                      | false       |
-| popper-class          | custom class name for TimePicker's dropdown              | ^[string]                                                                                       | —           |
-| range-separator       | range separator                                          | ^[string]                                                                                       | '-'         |
+| popper-class          | custom class name for TimePicker's dropdown              | ^[string]                                                                                       | ''          |
+| range-separator       | range separator                                          | ^[string]                                                                                       | -           |
```

Comment:
`-` Connectors also need to keep quotes for easy distinction

Reply:
updated